Examples/CaseStudies/SwiftUICaseStudies/04-Navigation-Lists-NavigateAndLoad.swift

```diff
@@ -73,25 +73,23 @@ struct NavigateAndLoadListView: View {
   @Bindable var store: StoreOf<NavigateAndLoadList>
 
   var body: some View {
-    WithViewStore(self.store, observe: { $0 }) { viewStore in
-      Form {
-        Section {
-          AboutView(readMe: readMe)
-        }
-        ForEach(viewStore.rows) { row in
-          NavigationLink(
-            "Load optional counter that starts from \(row.count)",
-            tag: row.id,
-            selection: viewStore.binding(
-              get: \.selection?.id,
-              send: { .setNavigation(selection: $0) }
-            )
-          ) {
-            IfLetStore(self.store.scope(state: \.selection?.value, action: \.counter)) {
-              CounterView(store: $0)
-            } else: {
-              ProgressView()
-            }
+    Form {
+      Section {
+        AboutView(readMe: readMe)
+      }
+      ForEach(store.rows) { row in
+        NavigationLink(
+          "Load optional counter that starts from \(row.count)",
+          tag: row.id,
+          selection: .init(
+            get: { store.selectedRowID },
+            set: { id in store.send(.setNavigation(selection: id)) }
+          )
```

Comment:
Can this use the new syntax?
```suggestion
          selection: $store.selectedRowID.sending(\.setNavigation)
```